docs/visual-testing/workflows/review.md

```diff
@@ -137,21 +137,23 @@ The user interface including the actions you can take are described in [User Int
 
 ## User Interface
 
-Selecting one of the builds allows you to get to the "Diff Review"-overlay, where you can Accept or Reject detected diffs.
+Selecting one of the builds allows you to get to the "Diff Review" page, where you can Accept or Reject detected diffs.
 
-<img src={useBaseUrl('/img/sauce-visual/DiffReviewPage.png')} alt="Diff Review-overlay"/>
+<img src={useBaseUrl('/img/sauce-visual/diff-review-page.jpg')} alt="Diff review page with toolbars"/>
 
-By default, you will see diffs in a side-by-side view, which lets you easily compare the current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
+By default, you will see diffs in a Side-by-side view, which lets you easily compare the Current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
 
-- Switch from side-by side-to focused view for easier comparison between the Baseline and Current snapshot.
+- Switch from Side-by-side to Overlay view for easier comparison between the Baseline and Current snapshot.
```

Comment:
It's now on the top of diff stage

Reply:
So maybe it should be removed from context of "On the right side"

Reply:
Good catch, thanks! I have updated it.